student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown', parent_permission = nil)
```

Comment:
- [ ] You did an excellent job by setting the default value for the name parameter to "Unkown" as requested. Unfortunately you forgot to change `nil` to `true`. Working once more on this, will ensure that you'll always abide and work based on the requirements provided by your client in future :smile: